docs/data/base/components/input/input.md

```diff
@@ -0,0 +1,58 @@
+---
+product: base
+title: React Input unstyled component and hook
+components: InputUnstyled
+githubLabel: 'component: input'
+packageName: '@mui/base'
+---
+
+# Unstyled input
+
+<p class="description">The `Input` component provides users with a field to enter and edit text.</p>
```

Comment:
Since the Markdown back ticks won't render inside the p tag, can we wrap it in `<code>` tags? I believe I saw that on another page recently.

```suggestion
<p class="description">The <code>Input</code> component provides users with a field to enter and edit text.</p>
```